### PR TITLE
Add trial info to init result

### PR DESCRIPTION
**Pull request by cpiehl1**
# Jira Issue: []

## What?
- Add trial info to the init widget result

## Why?
- Widget needs to know this

## Optional checklist
- [ ] Unit tests written
- [ ] Tested locally
- [ ] Codescouted

### CODE REVIEW

File: src/resolvers/partners/partners.graphqls [hunk 1]
```diff
@@ -26,4 +26,10 @@ type PartnerInitWidgetResult {
     market: Market!,
     partnerName: String!,
     partnerDefaultCampaignCode: String
+    trialInfo: TrialInfo
+}
+
+type TrialInfo {
+    startDate: LocalDate!
+    endDate: LocalDate!
 }
```
Comment (cpiehl1) [inline src/resolvers/partners/partners.graphqls:35]: I think we should be able to ignore the trial id here, can we also ignore the move in date?

**Comment by cpiehl1** [comment]
Lots of diff in the generated file, someone probably didn't run codegen before merging to master. But the only relevant changes here are in `partners.graphqls`.

File: src/resolvers/partners/partners.graphqls (hunk 1 above)
Comment (Bjoernbom) [reply src/resolvers/partners/partners.graphqls:35]: Yea I guess! 👍🏼

**Review by Bjoernbom — APPROVED** [review]

**Review by Bjoernbom — APPROVED** [review]

**Review by bystam — APPROVED** [review]
Looks good to me

**Pull request merged**